### PR TITLE
feat: Add CloudFormation Template

### PR DESCRIPTION
**Pull request by bryankaraffa**
### Description

Adds CloudFormation Template which can be used to create IAM Role and IAM Permission resources used by Flexera Automation Policies

Preview Docs/Template here: [./tools/cloudformation-template/](https://github.com/flexera-public/policy_templates/tree/feat/bk/add_cloudformation_template/tools/cloudformation-template)

### Contribution Check List

- [x] All tests pass.
- [X] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable
- [X] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: tools/cloudformation-template/README.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+     - `<AWS Account ID>` is the AWS Account ID of the account the CloudFormation Stack instance has been deployed to.
+     - `<IAM Role Name>` is the value of the *IAM Role Name* parameter provided to the CloudFormation StackSet.
+
+    For example, if the Stack instance was depoyed to AWS Account `123456789012` and the *IAM Role Name* parameter was `FlexeraAutomationPolicies-Org12345`, then the IAM Role ARN to input in Flexera Platform would be `arn:aws:iam::123456789012:role/FlexeraAutomationPolicies-Org12345`.
+
+    **See [Flexera Docs > Automation > AWS STS Multi-Account Credential Usage](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm#automationadmin_109256743_1136870) for more information.**
+
+</details>
+
```
Comment (taterdog590) [inline tools/cloudformation-template/README.md:102]: This section needs to be added or removed?

File: tools/cloudformation-template/FlexeraAutomationPolicies.template [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+  paramPermsAWSIdleComputeInstances:
+    Description: 'What permissions should policies using "AWS Idle Compute Instances" Policy Template be granted on the IAM Role that will be created?'
+    Type: String
+    Default: Read Only
+    AllowedValues:
+      - No Access
+      - Read Only
+
+Conditions:
+  # CreateExistingPolicyAttachment: !Not
```
Comment (taterdog590) [inline tools/cloudformation-template/FlexeraAutomationPolicies.template:131]: Are we leaving in these commented out sections?

**Review by taterdog590 — COMMENTED** [review]
Have some outstanding questions.

File: tools/cloudformation-template/FlexeraAutomationPolicies.template (hunk 2 above)
Comment (bryankaraffa) [reply tools/cloudformation-template/FlexeraAutomationPolicies.template]: Fixed!  We can add that functionality in a new release but removed the commented blocks for now

**Review by taterdog590 — APPROVED** [review]
LGTM

**Pull request merged**